src/Core/Domain/Address/Command/AddCustomerAddressCommand.php

```diff
@@ -147,14 +147,14 @@ public function __construct(
         string $city,
         int $countryId,
         string $postcode,
-        string $dni,
-        string $company,
-        string $vat_number,
-        string $address2,
-        int $id_state,
-        string $phone,
-        ?string $phone_mobile,
-        string $other
+        string $dni = null,
```

Comment:
Makes sense 🤔indeed these parameters are mandatory for a B2B business, but not a B2C business